src/components/Slider/Slider.tsx

```diff
@@ -56,6 +58,11 @@ const SliderItem = (props: SliderItemProps) => {
     );
     const { isFocusVisible, focusProps } = useFocusRing();
 
+    const handleLabelClick = (event: MouseEvent<HTMLLabelElement>) => {
+        event.preventDefault();
```

Comment:
From my point of view, I don't think the solution for this issue is to prevent the event, but rather check that `isFocusVisible` has the correct value.

Reply:
Label has been converted into a div so that default event does not need to be prevented